gstreamer-sdp/src/auto/m_i_k_e_y_message.rs

```diff
@@ -0,0 +1,166 @@
+        let data_len = data.len() as u16;
+        unsafe {
+            from_glib(ffi::gst_mikey_message_add_pke(self.to_glib_none_mut().0, C.to_glib(), data_len, data.to_glib_none().0))
+        }
+    }
+
+    pub fn add_rand(&mut self, rand: &[u8]) -> bool {
+        let len = rand.len() as u8;
+        unsafe {
+            from_glib(ffi::gst_mikey_message_add_rand(self.to_glib_none_mut().0, len, rand.to_glib_none().0))
+        }
+    }
+
+    pub fn add_rand_len(&mut self, len: u8) -> bool {
+        unsafe {
+            from_glib(ffi::gst_mikey_message_add_rand_len(self.to_glib_none_mut().0, len))
+        }
+    }
+
+    //pub fn add_t(&mut self, type_: MIKEYTSType, ts_value: &[u8]) -> bool {
+    //    unsafe { TODO: call ffi::gst_mikey_message_add_t() }
+    //}
+
+    pub fn add_t_now_ntp_utc(&mut self) -> bool {
+        unsafe {
+            from_glib(ffi::gst_mikey_message_add_t_now_ntp_utc(self.to_glib_none_mut().0))
+        }
+    }
+
+    pub fn base64_encode(&mut self) -> Option<String> {
+        unsafe {
+            from_glib_full(ffi::gst_mikey_message_base64_encode(self.to_glib_none_mut().0))
+        }
+    }
+
+    pub fn find_payload(&self, type_: MIKEYPayloadType, nth: u32) -> Option<MIKEYPayload> {
+        unsafe {
+            from_glib_none(ffi::gst_mikey_message_find_payload(self.to_glib_none().0, type_.to_glib(), nth))
+        }
+    }
+
+    //pub fn get_cs_srtp(&self, idx: u32) -> /*Ignored*/Option<MIKEYMapSRTP> {
+    //    unsafe { TODO: call ffi::gst_mikey_message_get_cs_srtp() }
+    //}
+
+    pub fn get_n_cs(&self) -> u32 {
+        unsafe {
+            ffi::gst_mikey_message_get_n_cs(self.to_glib_none().0)
+        }
+    }
+
+    pub fn get_n_payloads(&self) -> u32 {
+        unsafe {
+            ffi::gst_mikey_message_get_n_payloads(self.to_glib_none().0)
+        }
+    }
+
+    pub fn get_payload(&self, idx: u32) -> Option<MIKEYPayload> {
+        unsafe {
+            from_glib_none(ffi::gst_mikey_message_get_payload(self.to_glib_none().0, idx))
+        }
+    }
+
+    //pub fn insert_cs_srtp(&mut self, idx: i32, map: /*Ignored*/&MIKEYMapSRTP) -> bool {
+    //    unsafe { TODO: call ffi::gst_mikey_message_insert_cs_srtp() }
+    //}
+
+    pub fn remove_cs_srtp(&mut self, idx: i32) -> bool {
+        unsafe {
+            from_glib(ffi::gst_mikey_message_remove_cs_srtp(self.to_glib_none_mut().0, idx))
+        }
+    }
+
+    pub fn remove_payload(&mut self, idx: u32) -> bool {
+        unsafe {
+            from_glib(ffi::gst_mikey_message_remove_payload(self.to_glib_none_mut().0, idx))
+        }
+    }
+
+    //pub fn replace_cs_srtp(&mut self, idx: i32, map: /*Ignored*/&MIKEYMapSRTP) -> bool {
+    //    unsafe { TODO: call ffi::gst_mikey_message_replace_cs_srtp() }
+    //}
+
+    pub fn set_info(&mut self, version: u8, type_: MIKEYType, V: bool, prf_func: MIKEYPRFFunc, CSB_id: u32, map_type: MIKEYMapType) -> bool {
```

Comment:
There are all kinds of setters in this type but no getters, I assume some field accessors would be needed. But that's also for a future PR